jest-preset.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   testMatch: ['<rootDir>/src/**/__tests__/*.(ts|js)?(x)'],
   moduleFileExtensions: ['ts', 'tsx', 'js'],
   coveragePathIgnorePatterns: ['/node_modules/', '/__tests__/', 'index.ts', 'types.ts'],
-  collectCoverageFrom: ['src/**/*.{js,ts}'],
+  collectCoverageFrom: ['src/**/*.{ts,tsx}'],
```

Comment:
I'd keep `js` for backward compatibility. WDYT?

Reply:
we can assume that`.js` files is some kind of imported legacy code or 3rd party library which we do not want to test. All our code should be in a `.ts|.tsx` files IMO.

Reply:
Fair enough. Will merge it then.